rero_ils/modules/loans/api.py

```diff
@@ -289,18 +309,33 @@ def get_loans_by_patron_pid(patron_pid):
         yield Loan.get_record_by_pid(loan.pid)
 
 
-def patron_profile_loans(patron_pid):
+def patron_profile(patron):
     """Return formatted loans for patron profile display."""
     from ..items.api import Item
 
-    checkouts = []
+    patron_pid = patron.get('pid')
+
+    # Load organisation
+    patron_type_pid = patron.replace_refs()\
+        .get('patron_type', {}).get('pid')
+    patron_type = PatronType.get_record_by_pid(patron_type_pid)
+    if patron_type:
+        organisation_pid = patron_type\
+            .replace_refs().get('organisation', {}).get('pid')
+        organisation = Organisation.get_record_by_pid(organisation_pid)
```

Comment:
if all of these is to get the ogranisation, then you can just do this:
`organisation = Organisation.get_record_by_pid(patron.organisation_pid)`

Reply:
done